### ISSUE TITLE
Contrast

### ISSUE THREAD

**Issue by NJMannai** [dnet/androsphinx#16]
Device: 
- Nokia 7.1, Android 10, TalkBack 13.
- Pixel 4a, Android 12, TalkBack 13.

The search bar on the onboarding page, contains white (#FFFFF) letters on a yellow (#F6CE55) background. 
The contrast minimum for normal lettering is 4,5:1 and for large lettering 3:1. The contrast for this search bar is 1,5:1. 
This issue relates to all white lettering on yellow background.

Add a user screen: the contrast of 'master password' and 'confirm master password', seem too low in contrast. Due to security it wasn't possible to screenshot the page and look at it with the colour contrast analyser tool. 

Related to WCAG 1.4.3 Contrast (minimum)

**Comment by dnet** [comment]
Is there some webapp where I can look up the contrast ratio between two colors to experiment with alternative ones?

**Comment by NJMannai** [comment]
Absolutely,  we use https://www.tpgi.com/color-contrast-checker/, but you could also try https://webaim.org/resources/contrastchecker/ and there are many more, but not as accurate as CCA.
Best is, if you have access to the code you can see the color-codes there and check them with the CCA.

**Comment by dnet** [comment]
> The search bar on the onboarding page, contains white (#FFFFF) letters on a yellow (#F6CE55) background.
> The contrast minimum for normal lettering is 4,5:1 and for large lettering 3:1. The contrast for this search bar is 1,5:1.
> This issue relates to all white lettering on yellow background.

I changed the white letters to pitch black (`#000000`) in [83d52a2](https://github.com/dnet/androsphinx/commit/83d52a2d320cb02958e5d1f88493f05b098a746e), I hope this fixes the issue both there and in all other windows.

> Add a user screen: the contrast of 'master password' and 'confirm master password', seem too low in contrast. Due to security it wasn't possible to screenshot the page and look at it with the colour contrast analyser tool.

I disabled the security for those windows in a special build and measured it as `#666666` on `#FFFFFF`, the first tool you linked (CCA) says that's 5.7:1 which passes the 4,5:1 you mentioned above.

Is that acceptable, are we done with the contrast issues?

**Comment by NJMannai** [comment]
Hi András!

Sorry for the late reply, I was in a lot of meetings today.
That sounds very acceptable, the contrast ratio between the aforementioned black letters (#00000) on a lime green background (#83D52A) is 11.5:1.
Personally, It hurts my eyes a bit, but the contrast is good 😊
